autocti/charge_injection/plot/fit_ci_plotters.py

```diff
@@ -461,12 +461,10 @@ def figures_1d_data_binned(
 
         if columns_fpr:
             data = copy.copy(self.dataset.data)
-            y = data.apply_mask(mask=np.invert(fpr_mask)).binned_across_columns
+            y = data.apply_mask(mask=fpr_mask).binned_across_columns

```

Comment:
I don't really know what happened here but for some reason the test only passed without the inversion. If it looks dodgy I'll investigate further